ynr/apps/elections/uk/templates/includes/by-election-ctas.html

```diff
@@ -0,0 +1,73 @@
+{% if SHOW_BY_ELECTION_CTA %}
+    <h2>By-elections</h2>
+    <h3>Upcoming elections</h3>
+
+    <table>
+        <thead>
+        <tr>
+            <th>Election</th>
+            <th>Area</th>
+            <th>Candidates known</th>
+            <th>Locked?</th>
+        </tr>
+        </thead>
+        <tbody>
+            {% for pee in upcoming_pees %}
```

Comment:
I expect we're unlikely to hit this very often in real-world usage, but this could be made slightly more elegant by adding a case that hides whole table/headers in the case where `len(upcoming_pees) == 0` (and similarly below if `len(past_pees) == 0` ) instead of rendering a table with 0 rows.